Tests/BackendIntegrationTests/SignatureVerificationIntegrationTests.swift

```diff
@@ -92,6 +92,18 @@ class InformationalSignatureVerificationIntegrationTests: BaseSignatureVerificat
         expect(info.entitlements.verification) == .failed
     }
 
+    func testLogInWithValidSignature() async throws {
+        let info = try await Purchases.shared.logIn(UUID().uuidString).customerInfo
+        expect(info.entitlements.verification) == .verified
+    }
+
+    func testLogInWithInvalidSignature() async throws {
+        self.invalidSignature = true
+
+        let info = try await Purchases.shared.logIn(UUID().uuidString).customerInfo
+        expect(info.entitlements.verification) == .failed
+    }
```

Comment:
These exposed the fact that `LogInOperation` wasn't forwarding the verification result.